### DISCUSSION TITLE
To utilize the full power of Gatsby Preview, we are installing gatsby-preview-plugin to your site.

### DISCUSSION THREAD

**Discussion by friendglak** [gatsbyjs/gatsby#35742] [Help]
Hi, I get this error in Gatsby Cloud, even with preview disabled, any solution?

![image](https://user-images.githubusercontent.com/81096169/170163754-77cac643-938c-4a7d-9644-c8c49e55d640.png)

**Comment by LekoArts** [comment]
Hi!

Please contact https://www.gatsbyjs.com/support/ - thanks!
